lib/networkx/graph.rb

```diff
@@ -325,5 +325,13 @@ def edge_subgraph(edges)
         "received #{edges.class.name} instead."
       end
     end
+
+    def is_multigraph
+      self.class.name.include?('Multi')
+    end
+
+    def is_directed
+      self.class.name.include?('Di')
+    end
```

Comment:
This works, but why not have something more "safer" (not open-ended) like,

```ruby
[NetworkX::DiGraph, NetworkX::MultiDiGraph].include?(self.class)
```

Reply:
Sure.